### ISSUE TITLE
[Bug]Chart widget - On Mouse hover of custom chart fusion configuration the canvas slightly moves

### ISSUE THREAD

**Issue by YogeshJayaseelan** [appsmithorg/appsmith#5437]
## Description
Release Frozen - It is observed that On Mouse hover of custom chart fusion configuration the canvas slightly moves

**Expected Result:**
On mouse hovering on any text the canvas should not move

**Actual Result:**
On mouse hover of Custom Chart fusion configuration text the canvas slightly moves

**### Steps to reproduce the behaviour:**

1.Launch appsmith app in the release frozen environment 
2. Drag and drop chart widget 
3. Change the chart type to custom chart 
4. Mouse hover on the 'custom chart fusion configuration' and observe

**VideoLink:**
https://www.loom.com/share/bf025fe94f0745f68824c6241e2e0a4b

### Important Details

- Version: [14.1 (16611.1.21.161.3)]
- OS: [MacOS]
- Browser [chrome, safari]
- Environment [Release Frozen, release]

**Comment by somangshu** [comment]
@YogeshJayaseelan this issue is not related to the chart widget, it's the long helper which brings the scroll in canvas as it overflows.

**Comment by somangshu** [comment]
One possible solution could be reposition based on ref position and window size, [Check here](https://stackoverflow.com/a/51066294/5826265)

**Comment by YogeshJayaseelan** [comment]
> @YogeshJayaseelan this issue is not related to the chart widget, it's the long helper which brings the scroll in canvas as it overflows.

@somangshu Yes! noticed that the canvas slightly moves not only for chart and it is for all the the widgets tool tips when the property pane is placed near sides.

**Comment by riodeuno** [comment]
@YogeshJayaseelan I can't replicate this issue, can you verify this issue again?

**Comment by YogeshJayaseelan** [comment]
@riodeuno This happens only when the property pane is placed at the right side on the canvas and hovering on any of the longer tools tips 

Below is the one example 

https://www.loom.com/share/21ad3a0ab7c940f9bb022828ddb00dc1

**Comment by riodeuno** [comment]
I believe we can prevent overflow by assigning the tooltip's boundaries as viewport.

**Comment by ThisIsFaar** [comment]
hey @YogeshJayaseelan can I work on this issue?

**Comment by riodeuno** [comment]
@somangshu Yes, this is a relatively small fix based on a preliminary triage. 
@ThisIsFaar Thanks for offering to help with this issue. I'm assigning this to you. 

Note: You should find the code to the property pane in the `src/pages/Editor/PropertyPane` folder.
